lib/AuroraLip/Common/FormatDictionary_List.cs

```diff
@@ -233,6 +233,9 @@ public static partial class FormatDictionary
             //H.a.n.d.
             new FormatInfo(".fbc", FormatType.Archive, "Fables Chocobo archive", "H.a.n.d.") { Class= typeof(FBC)},
 
+            //Cing
+            new FormatInfo(".pac", "PCKG", FormatType.Archive, "Little King's Story Archive", "Cing"){ Class= typeof(PCKG_CING)},  // also pcha
```

Comment:
`new FormatInfo(".pac", "PCKG", FormatType.Archive, "Little King's Story Archive", "Cing"),`

if we use IMagicIdentify and IFileAccess the program will assign the class automatically.